apps/sms/js/thread_ui.js

```diff
@@ -1387,26 +1389,38 @@ var ThreadUI = {
    * markup to support bidirectional content, but other panels still use it with
    * mozL10n.setAttributes as it would contain only localizable text. We should
    * get rid of this method once bug 961572 and bug 1011085 are landed.
-   * @param {string|{ id: string, args: Object }} content Should be either safe
-   * HTML string or object with l10nId and l10nArgs.
+   * @param {string|{ raw: string }|{ html: string }|
```

Comment:
how about keeping "id" so that we always get an object as parameter? Or even "l10nId" so that it's really clear to read at the caller site